docs/io/votable/references.txt

```diff
@@ -1,6 +1,6 @@
 .. _BINARY: http://www.ivoa.net/documents/PR/VOTable/VOTable-20040322.html#ToC27
```

Comment:
Does it make sense to change this and all the other references to IVOA PRs in this section (i.e., also BINARY2 and TABLEDATA) to be to the appropriate RECs?  (It's not on the main topic of this pull request, of course; I could do a separate pull request on that after this one is merged.)

Reply:
Yes, I agree, but was leery of scope creep and rushing these in without at least sanity checking that the references are still appropriate.  A new PR makes sense to me.  Thanks for taking a look at this PR!